doc/api/util.md

```diff
@@ -14,6 +14,60 @@ it:
 const util = require('node:util');
 ```
 
+## `util.addSafeAbortSignalAbortListener(signal, resource)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `signal` {AbortSignal}
+* `listener` {Function|EventListener}
+* `options` {Object}
+  * `passive` {boolean} When `true`, serves as a hint that the listener will
+    not call the `Event` object's `preventDefault()` method.
+    **Default:** `false`.
+  * `signal` {AbortSignal} The listener will be removed when the given
+    AbortSignal object's `abort()` method is called.
+* Returns: {Disposable} that removes the `abort` listener.
```

Comment:
Do we need to return an object that is currently not standardized yet? Can we return something that we can control?

Reply:
>  Can we return something that we can control?

I'm also not sure how the docs should look and I'm not sure "disposable" is the name (is it?) but it's basically "something that has a Symbol.dispose".

Unlike AbortSignal/EventTarget - disposables are just a function call - so we control this.

Reply:
@benjamingr that is the name the spec calls it:
https://github.com/tc39/proposal-explicit-resource-management/blob/3b576c0bb756fcbb4b91c5556c3609806f9abaf9/README.md?plain=1#L1164-L1180